src/pages/settings/ProfilePage.js

```diff
@@ -1,24 +1,254 @@
+                                    />
+                                </View>
+                                {this.state.pronouns === CONST.PRONOUNS.SELF_SELECT && (
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.selfSelectedPronouns}
+                                        onChangeText={selfSelectedPronouns => this.setState({selfSelectedPronouns})}
+                                        placeholder="Self-select your pronoun"
+                                    />
+                                )}
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>
+                                    {Str.isSMSLogin(this.props.myPersonalDetails.login)
+                                        ? 'Phone Number' : 'Email Address'}
+                                </Text>
+                                <TextInput
+                                    style={[styles.textInput, styles.disabledTextInput]}
+                                    value={Str.isSMSLogin(this.props.myPersonalDetails.login)
+                                        ? Str.removeSMSDomain(this.props.myPersonalDetails.login)
+                                        : this.props.myPersonalDetails.login}
+                                    editable={false}
+                                />
+                            </View>
+                            <View style={styles.mb2}>
+                                <Text style={[styles.mb1, styles.formLabel]}>Timezone</Text>
+                                <RNPickerSelect
+                                    onValueChange={selectedTimezone => this.setState({selectedTimezone})}
+                                    items={timezones}
+                                    style={this.state.isAutomaticTimezone ? {
+                                        ...styles.picker,
+                                        inputIOS: [styles.picker.inputIOS, styles.textInput, styles.disabledTextInput],
+                                        inputAndroid: [
+                                            styles.picker.inputAndroid, styles.textInput, styles.disabledTextInput,
+                                        ],
+                                        inputWeb: [styles.picker.inputWeb, styles.textInput, styles.disabledTextInput],
+                                    } : styles.picker}
+                                    useNativeAndroidPickerStyle={false}
+                                    value={this.state.selectedTimezone}
+                                    Icon={() => <Icon src={DownArrow} />}
+                                    disabled={this.state.isAutomaticTimezone}
+                                />
+                            </View>
+                            <Checkbox
```

Comment:
Minor comment: can we add just a little bit more margin between the text input and the checkbox row? Either by adding `mtXXXX` to the checkbox or `mbXXXX` to the input, where XXXX is something like 2 or 3.

Reply:
bumped it up to a 3
<img width="494" alt="Screen Shot 2021-03-17 at 6 31 21 AM" src="https://user-images.githubusercontent.com/12075600/111475570-73e83880-86ea-11eb-8507-2de78356a318.png">

Reply:
Nice, looks great! Nice and fluffy spacing throughout.